docs/2.0/extensions/mentions.md

```diff
@@ -35,25 +35,25 @@ $config = [
         // Sample Input:  `@colinodell`
         // Sample Output: `<a href="https://www.github.com/colinodell">@colinodell</a>`
         'github_handle' => [
-            'symbol'    => '@',
+            'prefix'    => '@',
             'regex'     => '/^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}(?!\w)/',
```

Comment:
Good catch!  We also need to remove the starting `/^` and trailing `/` from all the regexes.

Reply:
Ah, right... done.

Reply:
This almost makes me wonder if it should be called `pattern` now instead of `regex`?

Reply:
I like that :)

Reply:
That can be a follow-up. I don't have time to include that in this PR.

Reply:
I've created https://github.com/thephpleague/commonmark/issues/569 to track that